lib/locales/da-DK.yml

```
@@ -565,3 +565,5 @@ da-DK:
           - Bænk
           - Ur
           - Pung
+    vehicle:
+      license_plate: '[A-Z]{2}[0-9]{5}'
```

Sorry, I just remembered that the generator for `license_plate` uses the `numerify` and `letterify` helpers by default:
[https://github.com/faker-ruby/faker/blob/46130d9212932c5ad3e5523413ef2189937f55af/lib/faker.rb#L53 and ](https://github.com/faker-ruby/faker/blob/46130d9212932c5ad3e5523413ef2189937f55af/lib/faker.rb#L64)

The other locale files use these helpers for generating random numbers and letters (including the vehicle english version). Let's follow the pattern for consistency:

```suggestion
    vehicle:
      license_plate: '??#####'
```

---

hi @woller just checking in on the comment above. Apologies for missing the helper in the first place. Thank you!